src/components/GameListItem.vue

```diff
@@ -2,7 +2,12 @@
   <div>
     <v-row class="list-item" data-cy="game-list-item">
       <v-col v-if="$vuetify.display.lgAndUp" sm="2" lg="3">
-        <v-img src="/img/logo-head.svg" class="my-1" contain :height="$vuetify.display.smAndUp ? 62 : 32" />
+        <v-img
+          src="/img/logo-head.svg"
+          class="my-1"
+          cover
+          :width="62"
```

Comment:
`height` is no longer supported, so moved these sorts of things to width. Eventually we should move these to CSS imo.

Reply:
agreed